promql/bench_test.go

```diff
@@ -5,7 +5,7 @@
 //
 // http://www.apache.org/licenses/LICENSE-2.0
 //
-// Unless required by applicable law or agreed to in writing, softwar
+// Unless required by applicable law or agreed to in writing, software
```

Comment:
This is odd. Generally lawyers get annoyed at things like this.

Reply:
This correction looks fine.